api/api-doc/storage_service.json

```diff
@@ -881,6 +881,14 @@
                           "allowMultiple":false,
                           "type":"string",
                           "paramType":"query"
+                      },
+                      {
+                          "name":"scope",
+                          "description":"Stream phase scope",
+                          "required":false,
+                          "allowMultiple":false,
+                          "type":"string",
+                          "paramType":"query"
```

Comment:
it's allowed to add "enum" property to a parameter, like
```yaml
{
    "name":"scope",
    "description":"Stream phase scope",
    "required":false,
    "allowMultiple":false,
    "type":"string",
    "paramType":"query",
    "enum": ["all", "dc", "rack", "node"]
}
```

`seastar-json2code.py` represents it as an `enum`, it also generates a parser which converts a string to this enum.